### PR TITLE
Add review page accordions

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**

The review page now has an accordion with an item for each page that matches the desired design. The report still needs to be marked as `Submitted` in the backend after saving of a report (https://github.com/HHS/Head-Start-TTADP/issues/98) is closed.

 * Each page defines sections that are used in the review accordion
 * Simplified how the activity report imports the pages for the report
 * Date picker now accepts a string MM/DD/YYYY and spits out the same
 * Activity report page file names are no longer capitalized because they are no longer react components

**How to test**

Sandbox:

Test on https://tta-smarthub-sandbox.app.cloud.gov/activity-reports/review. Note the "approvers" section isn't working correctly because the seed files aren't being loaded in for this branch. Once https://github.com/adhocteam/Head-Start-TTADP/pull/78 is merged the approvers select box will work.

Local:

1. Pull down
2. Run migrations and seeders
3. Go to Activity report and fill out a report
4. Click the accordion in the review page, verify form values are correctly shown

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/135

**Checklist**

- [ ] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: frontend/src/components/Navigator/components/Form.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ function Form({
   return (
     <UswdsForm onSubmit={handleSubmit(onContinue)} className="smart-hub--form-large">
       {renderForm(hookForm)}
-      <Button className="stepper-button" type="submit" disabled={!formState.isValid}>Continue</Button>
+      <Button type="submit" disabled={!formState.isValid}>Continue</Button>
```
Comment (jasalisbury) [inline frontend/src/components/Navigator/components/Form.js:54]: Removing CSS class that was doing nothing, missed it in a previous PR

File: frontend/src/components/Navigator/components/SideNav.css [hunk 2]
```diff
@@ -80,3 +80,11 @@
...
   transition: 0.2s ease-in-out;
 }
+
+.smart-hub--navigator-item:first-child .smart-hub--navigator-link-active {
+  border-top-right-radius: 4px;
+}
+
+.smart-hub--navigator-item:last-child .smart-hub--navigator-link-active {
+  border-bottom-right-radius: 4px;
+}
```
Comment (jasalisbury) [inline frontend/src/components/Navigator/components/SideNav.css:90]: Fixing slight style issue on the navigator's side nav where the border radius is squared when the first or last element is selected

File: frontend/src/pages/ActivityReport/Pages/activitySummary.js [hunk 3]
```diff
@@ -307,4 +311,73 @@ ActivitySummary.propTypes = {
   control: PropTypes.object.isRequired,
 };
 
-export default ActivitySummary;
+const sections = [
```
Comment (jasalisbury) [inline frontend/src/pages/ActivityReport/Pages/activitySummary.js:314]: There appears to be the potential for defining the whole form in an object similar to this one and simplifying the ~300 lines of code above, but that would be a bigger project then we have the capacity to tackle at this point.

**Comment by kryswisnaskas** [comment]
The accordion and the summary look great! 🚀 

At some point (for a different PR) I can see a request to preserve the expanded state on the accordion when navigating away and coming back.

Minor item on this PR. Can we add a bit more space between the label and the textfield for the managers'
image

**Review by kryswisnaskas — APPROVED** [review]
👍

**Pull request merged**